src/sidebar/components/moderation/test/ModerationControl-test.js

```diff
@@ -0,0 +1,75 @@
+import { mockImportedComponents, mount } from '@hypothesis/frontend-testing';
+import sinon from 'sinon';
+
+import { defaultAnnotation } from '../../../test/annotation-fixtures';
+import ModerationControl, { $imports } from '../ModerationControl';
+
+describe('ModerationControl', () => {
+  let fakeProfile;
+  let fakeFocusedGroup;
+  let fakeStore;
+
+  beforeEach(() => {
+    fakeProfile = { userid: '123' };
+    fakeFocusedGroup = { pre_moderated: true };
+    fakeStore = {
+      profile: sinon.stub().returns(fakeProfile),
+      focusedGroup: sinon.stub().returns(fakeFocusedGroup),
+    };
+
+    $imports.$mock(mockImportedComponents());
+    $imports.$mock({
+      '../../store': {
+        useSidebarStore: () => fakeStore,
+      },
+    });
+  });
+
+  function createComponent(annotation) {
+    return mount(<ModerationControl annotation={annotation} />);
+  }
+
+  context('when current user is not a moderator', () => {
```

Comment:
Strictly speaking, all the client knows is that the user cannot moderate the annotation, but it might not be worth making that distinction here.

Reply:
We could probably reword it to "when annotation can be moderated" and "when annotation cannot be moderated", to be more accurate.